packages/calypso-products/src/plans-list.tsx

```diff
@@ -1764,6 +1817,21 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 		getStoreSlug: () => PLAN_JETPACK_COMPLETE,
 		getPathSlug: () => 'complete',
 		getProductId: () => 2014,
+		getJetpackIncludesInfo: () => [
+			JETPACK_INCLUDES_REAL_TIME_BACKUP,
+			JETPACK_T2_INCLUDES_CLOUD_STORAGE,
+			JETPACK_YEARLY_INCLUDES_LOG_ARCHIVE,
+			JETPACK_YEARLY_INCLUDES_RESTORES,
+			JETPACK_INCLUDES_REALTIME_MALWARE_SCANNING,
+			JETPACK_COMPLETE_INCLUDES_SPAM_PROTECTION,
+			...JETPACK_COMPLETE_INCLUDES_ADDITIONAL,
+		],
+		getJetpackBenefits: () => [
+			...COMMON_JETPACK_PLAN_BENEFITS,
+			...JETPACK_COMPLETE_ADDITIONAL_BENEFITS,
+			JETPACK_BENEFITS_BEST_IN_CLASS_SUPPORT,
+		],
+		getJetpackRecommendedFor: () => COMMON_JETPACK_RECOMMENDED_FOR,
```

Comment:
Any specific reason for adding `Jetpack` in function names? Why not simply `getBenefits` instead of `getJetpackBenefits`?

Also, `getJetpackIncludesInfo` can be `getWhatIsIncluded` and likewise `includesInfo` prop on the product can be `whatIsIncluded`